server/src/main/java/org/elasticsearch/search/aggregations/bucket/range/InternalRange.java

```diff
@@ -156,7 +171,13 @@ private static String generateKey(double from, double to, DocValueFormat format)
         public void writeTo(StreamOutput out) throws IOException {
             out.writeString(key);
             out.writeDouble(from);
+            if (out.getVersion().onOrAfter(Version.V_8_0_0)) {
```

Comment:
If you merge #82435 first, you can change this value to `V_7_1_7` before merging this PR